alvr/filesystem-layout/src/lib.rs

```diff
@@ -25,9 +25,12 @@ impl Layout {
         if cfg!(windows) {
             self.openvr_driver_dir
                 .join("bin/win64/driver_alvr_server.dll")
-        } else {
+        } else if cfg!(target_os = "linux") {
             self.openvr_driver_dir
                 .join("bin/linux64/driver_alvr_server.so")
+        } else {
+            self.openvr_driver_dir
+                .join("bin/macos/driver_alvr_server.dylib")
```

Comment:
Are you sure this is where SteamVR wants this? I'd expect it to follow the same format as the other platforms

Reply:
macos support is dummy. I don't plan to make it work on macos but at least I want xtask commands to succeed and work predictably.